packages/v4/patternfly-docs/pages/home.css

```diff
@@ -135,6 +135,13 @@
   background-image: url("../images/desktop.svg");
 }
 
+img.pf-v5-c-image__light-showcase {
+  height: 100%;
+  width: auto;
+  max-width: 100%;
+  transform: translateX(10%);
```

Comment:
I removed this in favor of Lucia's suggestion that we remove the image in smaller / mobile viewports

> For resizing, is there any way to fix the height and just have the browser cut off the right side of the image as the viewport shrinks? And then in tablet/mobile just remove the component showcase image all together?

I've been investigating and it seems like styling this image as a background image could be a good idea to prevent it from shrinking in the browser... WDYT @mattnolting ?